srcpkgs/incus/files/incus/run

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 exec 2>&1
 [ -r conf ] && . ./conf
-exec /usr/libexec/incus/incusd --group _incus-admin --syslog ${OPTS:- --verbose}
```

Comment:
What is this change about?

Reply:
services shouldn't log directly to syslog, but to stdout, where runit will collect the logs

Reply:
thanks for explaining